File: solutions/ruby/2022/01/main.rb
# frozen_string_literal: true

require 'bundler/setup'
require_relative 'lib/solutions'

INPUT = File.read File.join(__dir__, '../../../../inputs/2022/01/input')

puts "[2022/01/1] #{Solutions202201.part_1 INPUT}"
puts "[2022/01/2] #{Solutions202201.part_2 INPUT}"

File: solutions/elixir/2022/01/main.exs
Code.require_file("lib/solutions.exs", __DIR__)

input = File.read!(Path.join([__DIR__, "../../../../inputs/2022/01/input"]))

IO.puts(~s"[2022/01/1] #{Solutions202201.part_1(input)}")
IO.puts(~s"[2022/01/2] #{Solutions202201.part_2(input)}")
